test/read_only_api_tests.jl

```diff
@@ -42,7 +42,7 @@ auth = authenticate(string(circshift(["bcc", "3fc", "03a", "33e",
     @test hasghobj(ghjl, first(repos(julweb; auth = auth)))
 
     # test sshkey/gpgkey retrieval
-    @test GitHub.sshkeys(testuser; auth = auth)[1][1]["key"] == testuser_sshkey
+    @test_broken GitHub.sshkeys(testuser; auth = auth)[1][1]["key"] == testuser_sshkey
```

Comment:
https://github.com/julia-github-test-bot.keys no longer appears to work.

Reply:
I think keys are automatically removed if they haven't been used in some time (1 year I think).